wp-content/themes/pub/wporg-learn-2020/template-parts/component-series-navigation.php

```diff
@@ -0,0 +1,46 @@
+<?php
+$series_term = wporg_workshop_series_get_term( $post );
+$previous    = wporg_workshop_series_get_adjacent( 'previous', $post );
+$next        = wporg_workshop_series_get_adjacent( 'next', $post );
+?>
+<nav class="workshop-series">
+	<h2>
+		<?php
+		printf(
+			wp_kses_post( __( 'Series: %s', 'wporg-learn' ) ),
+			sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_html( get_term_link( $series_term ) ),
+				esc_html( $series_term->name )
+			)
+		);
+		?>
+	</h2>
+	<?php echo wp_kses_post( wpautop( term_description( $series_term->term_id ) ) ); ?>
+	<ul class="row video-grid">
+		<li class="col-6 video-grid_item">
+			<?php if ( $previous ) : ?>
+				<?php esc_html_e( 'Previous workshop:', 'wporg-learn' ); ?>
+				<a class="video-grid_item_link" href="<?php echo esc_url( get_the_permalink( $previous ) ); ?>">
+					<?php
+					// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+					echo wporg_get_post_thumbnail( $previous, 'medium' );
+					?>
+					<?php echo wp_kses_post( get_the_title( $previous ) ); ?>
+				</a>
+			<?php endif; ?>
+		</li>
+		<li class="col-6 video-grid_item">
+			<?php if ( $next ) : ?>
```

Comment:
Same here, outputting the `li` creates an empty space that messes with the layout.

Reply:
I understand that we are leading them with right (next) and left (previous). But I didn't make the connection right away. Maybe they are too big? A second opinion may make sense.

Reply:
👍 Yeah, I was kind of wondering if they were too big. Thoughts @melchoyce ?

Reply:
I agree - it feels like the next/previous navigation is too big and therefore unclear. We don't need to show the thumbnails here - maybe large arrow blocks with the title of the workshops in them instead?